src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -1377,6 +1382,36 @@ def GetTile(x: int, y: int) -> PyGameObject:
     """
     pass
 
+def GetStaticsAt(x: int, y: int) -> list[Any]:
```

Ruff (0.12.2)

Undefined name `Any`

(F821)